### PR TITLE
network: change get_netstack interface.

### PR DESCRIPTION
**Pull request by pillip8282**
get_netstack_byfd is added
A function get_netstack() provides 2 interfaces(user domain socket, bsd
socket(lwip)). Those can be distinguished by file descriptor or the
request fo caller. therefore get_netstack should be provided by those
parameters.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f6933384465df4559cc7db24aaea5377b8b7e1e6] - Code Rule Check Result:
os/net/netmgr/netstack.c:50: ERROR: [BRC_M_SMT] else should follow close brace '}'
os/net/netmgr/netstack.c:65: ERROR: [BRC_M_SMT] else should follow close brace '}'

**Comment by seinfra** [comment]
Target : [f6933384465df4559cc7db24aaea5377b8b7e1e6] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [89e3ca68b2f191f06f4f714fd22b58819ed5fc1e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [89e3ca68b2f191f06f4f714fd22b58819ed5fc1e] - Code Rule Check Result:
os/net/netmgr/netstack.c:71: ERROR: [BRC_M_SMT] else should follow close brace '}'

**Comment by seinfra** [comment]
Target : [3e0470b6865e56e00e59ca3e0366ec165c8de063] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [3e0470b6865e56e00e59ca3e0366ec165c8de063] - Code Rule Check (C++) OK.

File: os/net/netmgr/netstack_uds.c [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+	return 0;
+}
+
+static ssize_t uds_sendmsg(int sockfd, struct msghdr *msg, int flags)
+{
+	return 0;
+}
+
+
+struct netstack_ops g_uds_stack_ops = {
```
Comment (junmin-kim) [inline os/net/netmgr/netstack_uds.c]: Would you set it as all NULL and remove uds_xxx? I'll connect them.

Comment (pillip8282) [reply os/net/netmgr/netstack_uds.c]: You can modify it after it is merged

**Comment by sangam-swami** [comment]
> or the
> request fo caller.

 or the
request to caller ?

**Comment by pillip8282** [comment]
sangam-swami
requests from callers

**Comment by seinfra** [comment]
Target : [0e34721ce535634113d58fede6945dde9b9015ba] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0e34721ce535634113d58fede6945dde9b9015ba] - Code Rule Check OK.

**Pull request merged**